src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,179 @@
+					<div class="space-y-6 h-full px-5 pt-6 pb-10 overflow-y-auto">
+						<ul class="mt-2 text-left">
+							{#each $results as article}
+								{@const highlight = article?._highlightResult}
+								{#if highlight?.content_excerpt.matchedWords.length > 0}
+									<li class="px-3 py-2.5 group cursor-pointer transition">
+										<a href={`/${article.slug}`}>
+											{#if highlight.title.matchedWords.length > 0}
+												<p
+													class="text-[18px] font-powerGroteskBold font-bold line-clamp-1 transition [&>em]:text-[#0CDEE9] [&>em]:font-medium [&>em]:!not-italic"
+												>
+													{@html highlight.title.value}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)